.chloggen/remove-promtail-receiver.yaml

```diff
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'deprecation'
```

Comment:
I'm not sure what change_type this PR lies to... deprecation fits the most probably, but I can change it